epxresults/run.py

```diff
@@ -432,6 +434,64 @@ def get_table_variable(
 
         return pd.Series(d, name=variable)
 
+    def get_list_table_variable(
+        self,
+        variable: str,
+        sim_day: int = None,
+        long: bool = False,
```

Comment:
I think we should use the same parameter name that is used in the FREDJob class methods:
`format : str, optional
            the default DataFrame format, either 'wide' or 'long'. By default,
            long format DataFrames are returned.`